packages/adapter-static/index.js

```diff
@@ -1,5 +1,13 @@
+import { createReadStream, createWriteStream, statSync } from 'fs';
+import { pipeline } from 'stream';
+import glob from 'tiny-glob';
+import { promisify } from 'util';
+import zlib from 'zlib';
+
+const pipe = promisify(pipeline);
+
 /** @type {import('.')} */
-export default function ({ pages = 'build', assets = pages, fallback } = {}) {
+export default function ({ pages = 'build', assets = pages, fallback, precompress = true } = {}) {
```

Comment:
it should continue to default to `false`, i think

```suggestion
export default function ({ pages = 'build', assets = pages, fallback, precompress = false } = {}) {
```